lib/TokenManager.js

```diff
@@ -0,0 +1,84 @@
+var util = require('./util');
+var AuthSdkError = require('./errors/AuthSdkError');
+var config = require('./config.json');
+var cookies = require('./cookies');
+
+function tokenStorageBuilder(storage) {
```

Comment:
One thing we might want to think about in the future is extracting this storage logic to another util file so that we can use it for the /authn flow (i.e. to store the stateToken).